Dockerfile

```diff
@@ -3,17 +3,24 @@
 ARG BASE_IMG_TAG=nonroot 
 
 ## Build Image
-FROM golang:1.18-bullseye as build
+FROM golang:1.18.2-alpine3.15 as build
+
+RUN set -eux; apk add --no-cache ca-certificates build-base;
+
```

Comment:
`LINK_STATICALLY=true` is needed to pick up certain build flags needed for statically linking cowm wasm dependencies

`RUN set -eux; apk add --no-cache ca-certificates build-base;` is needed because we changes the base image and need to install some build tools for building cosm wasm dependencies.

This was adapted from the latest image in wasmd repository: https://github.com/CosmWasm/wasmd/blob/2b0b1677df42daffa9817cf2f4e266f4e40d69b8/Dockerfile#L9